ConfCore/UserDataSyncEngine.swift

```diff
@@ -442,13 +442,13 @@ public final class UserDataSyncEngine {
         os_log("Will commit %{public}d changed record(s) to the database", log: log, type: .info, records.count)
 
         performRealmOperations { realm in
-            records.forEach { record in
```

Comment:
<img width="1458" alt="screen shot 2018-06-11 at 2 17 08 pm" src="https://user-images.githubusercontent.com/8225090/41252240-2bf17d18-6d82-11e8-90ac-9c9d31899e11.png">

Reply:
what version are you using?

Reply:
Xcode 9.4. This file doesn't get compiled unless you're on an iCloud scheme